wire-ios/Wire-iOS/Sources/Helpers/UIBarButtonItem+StyleKit.swift

```diff
@@ -38,21 +37,6 @@ extension UIBarButtonItem {
         )
     }
 
-    func setIcon(_ icon: StyleKitIcon) {
-        image = icon.makeImage(
-            size: .tiny,
-            color: IconColors.foregroundDefaultBlack
-        )
-    }
-
-    static func createCloseItem() -> UIBarButtonItem {
-        let item = UIBarButtonItem(icon: .cross, target: nil, action: nil)
```

Comment:
Could the `.cross` StyleKit icon already be removed or is it still used somewhere else?